flambda-backend/testsuite/tests/asmgen/quicksort.cmm

```diff
@@ -2,8 +2,12 @@
  readonly_files = "main.c asan_report_wrappers.c";
  arguments = "-DSORT -DFUN=quicksort main.c asan_report_wrappers.c";
  asmgen;
+ skip;
+ reason="Needs rewriting to not use Cassign";
```

Comment:
You have to put `skip` before any meaningful action if you want it to work properly (and you probably want to set the reason before skipping too, although that is less important).

Reply:
Should be fixed